docs/website/docs/user-guides/quickstart/docs-mdx/go/go_odo_dev_output.mdx

```diff
@@ -23,5 +23,4 @@ $ odo dev
  Keyboard Commands:
```

Comment:
Can you change `/home/user/quickstart-demo/go-demo` to `/home/user/quickstart-demo` and for all the *_odo_dev_output.mdx files? It is inline with some changes I have made in #6511.
You'll also need to make changes in quickstart test.